gemmbench/gemm_bench.py

```diff
@@ -38,7 +38,7 @@ def compile_gemm(tag, config, kernel_dir, vmfb_dir, target, extra_compiler_args,
         help="Set the logging level",
     )
 
-    parser.add_argument("--target", help="The IREE hip target to compile for", type=str, default="gfx942")
+    parser.add_argument("--target", help="The IREE hip target to compile for. The special value host_cpu results in a llvm-cpu benchmark instead of HIP, compiled for the host CPU.", type=str, default="gfx942")
```

Comment:
Since we are inventing a new name here anyway, maybe just call this 'cpu'?

Reply:
Because in addition of specifying the CPU as a target architecture, it goes further and specified the host CPU as the CPU to optimize for. The resulting vfmb's take advantage of the features of the host CPU, but would crash if run on another CPU lacking the same features.